lib/paper_trail/model_config.rb

```diff
@@ -122,6 +122,12 @@ def setup(options = {})
       setup_callbacks_from_options options[:on]
     end
 
+    # Single source of truth of the primary_key used for the versions association
```

Comment:
Please add comment about how this method will be used by other gem, referencing all issue and PR numbers.

Also, `@api public` would be appropriate here.

Reply:
I thought this was `private` because users of the `paper_trail` weren't going to override it. Anyways, I changed it

Reply:
> I thought this was private because users of the paper_trail weren't going to use it. Anyways, I changed it

I prefer public because I see your new gem as a user of PT, and I want to treat changes to this method as breaking changes.